tests/acceptance/features/apiOcm/deleteFederatedConnections.feature

```diff
@@ -131,3 +131,26 @@ Feature: delete federated connections
     Given user "Brian" has deleted federated connection with user "Alice"
     When user "Brian" tries to delete federated connection with user "Alice" using the Graph API
     Then the HTTP status code should be "404"
+
+  @issue-10223
+  Scenario Outline: local user tries to delete previously deleted federated connection with random idp
+    Given using server "LOCAL"
+    And user "Alice" has deleted federated connection with user "Brian"
+    When user "Alice" tries to delete federated connection with user "Brian" with idp "<idp>" using the Graph API
+    Then the HTTP status code should be "400"
+    Examples:
+      | idp            |
+      | localhost:9244 |
+      | localhost:123  |
+      | localhost:11   |
+
+  @issue-10223
+  Scenario Outline: federated user tries to delete previously deleted federated connection with random idp
+    Given user "Brian" has deleted federated connection with user "Alice"
+    When user "Brian" tries to delete federated connection with user "Alice" with idp "<idp>" using the Graph API
+    Then the HTTP status code should be "400"
+    Examples:
+      | idp            |
+      | localhost:9244 |
+      | localhost:123  |
+      | localhost:11   |
```

Comment:
do we need check all combination? IMO, checking one problematic value should be enough. If each of them show different behaviour then only having all makes sense.
```suggestion
  Scenario Outline: federated user tries to delete previously deleted federated connection with random idp
    Given user "Brian" has deleted federated connection with user "Alice"
    When user "Brian" tries to delete federated connection with user "Alice" with idp "<idp>" using the Graph API
    Then the HTTP status code should be "400"
    Examples:
      | idp            |
      | localhost:9244 |
      | localhost:123  |
      | localhost:11   |
```